### DISCUSSION TITLE
Group Assignment #3 - Revised Memos📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#16] [Assignments]
### Due 5/4

Feel free to post your revised memo here or email it to me when you would like me to review it.

### Please submit your surveys via this link (NOT THIS THREAD!):

- https://forms.gle/yrKyrHaoAJoW9xbbA

**Comment by amandatlxu** [comment]
Federally Qualified Healthcare Centers: https://docs.google.com/document/d/1ydP-jpP1JHqKkHpxV2v1MpcVio6JViphi8Da7DibRXg/edit?usp=sharing

**Comment by paige-lee** [comment]
https://docs.google.com/document/d/16H78ls48ZjxNmxPv0JnZETl5wAMgEl2U8YYFAhhklPU/edit?usp=sharing

**Comment by shintuzd** [comment]
https://docs.google.com/document/d/1FT8f9JGi9b3lmb5S8jR_yNAL9k9yCb7yiGSRtZOYTqY/edit#

**Comment by emai14** [comment]
Educational Attainment Revised Memo: https://docs.google.com/document/d/1LzpjFhMLLv56jv7UYGkgEMmND9HpxJgBzGC8izY1hEc/edit#heading=h.unwtcohi2eir

**Comment by danigold1020** [comment]
Ocean Pollution (Oil Spills): [https://docs.google.com/document/d/1zgN4oBwbOEASws4noyHNEINXmB_a6VSdyEVYBMFaF5U/edit#]( https://docs.google.com/document/d/1zgN4oBwbOEASws4noyHNEINXmB_a6VSdyEVYBMFaF5U/edit#)

**Comment by cassidytu** [comment]
gender inequality in transportation: https://docs.google.com/document/d/1olcr3Z4hpixXaZKbzb3oDmfF1uPUQZj2akojj_KKlI4/edit?usp=sharing

**Comment by czhangy** [comment]
https://docs.google.com/document/d/1SlxKN--ncXRuaQKxXF002cXKd_axPdxvhcRCj5rBJLs/edit?usp=sharing

**Comment by mandymwlai** [comment]
https://docs.google.com/document/d/1l9W414V8UDmaLfZBaI6EVYt5nhL2Q27mAYzRwXqQ_4o/edit?usp=sharing
